00-Login/README.md

```diff
@@ -28,32 +27,18 @@ You should see the `A0Auth0` pod being installed and linked to the sample app.
 
 1. Open the [Auth0 dashboard](https://manage.auth0.com/dashboard/) and click **Applications.**
 2. Select your existing Application from the list or click **Create Application** at the top to create a new one of type **Native**. 
-3. On the **Settings** tab for the Application, copy the "Client ID" and "Domain" values and paste them in the `app/auth0-credentials.js` file. This file is being used in the `app/index.js` file to instantiate the library. You can also change and do that manually this way:
-
-```js
-import Auth0 from 'react-native-auth0';
+3. On the **Settings** tab for the Application, copy the "Client ID" and "Domain" values and paste them in the `app/auth0-credentials.js` file. This file is being used in the `app/index.js` file to instantiate the library.
+4. In the **Allowed Callback URLs** field, paste in the text below and replace `YOUR_DOMAIN` with the Domain from step 3. These URLs are required for the authentication result to be redirected from the browser to the app:
 
-const config = {
-  domain: 'YOUR_DOMAIN',
-  clientId: 'YOUR_CLIENT_ID'
-}
-
-const auth0 = new Auth0(config);
+```
+com.auth0samples://YOUR_DOMAIN/ios/com.auth0samples/callback, 
+com.auth0samples://YOUR_DOMAIN/android/com.auth0samples/callback
 ```
 
-### Deep linking setup
-The methods required to handle browser deep linking are already in place. If you need to review them, see the [React Native QuickStart](https://auth0.com/docs/quickstart/native/react-native/00-login).
-
-The URL that the Browser app calls to pass the authentication result back to your app is created using a combination of the following values:
-
-- package name / product bundle identifier; currently `com.auth0samples` on both platforms.
-- platform name
-- auth0 domain
-
-If you happen to change the first remember to take note of it to [update the callback URLs](#whitelist-the-urls) later.
-
+5. Add the same values to the **Allowed Logout URLs** field as well. These are required for the browser to redirect back to the app after the user logs out.
+6. Scroll down and click **Save Changes**.
 
-#### Additional step for Android apps
+### Deep linking setup (Android only)
```

Comment:
Do you think this is better than what we had before? For me, it reads like we are not showing how to set up deep-linking on iOS. Also, we have already used once the "Additional setup for X apps" when we instructed to run `pod install`.

Reply:
I think all of the information about deep linking in this sample was not necessary. Sample install steps are not the place to talk through features. Otherwise, where would we stop? 

Good call on the other additional steps ... that didn't occur to me. Will commit.